Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView_Tests.swift

```diff
@@ -10,27 +10,39 @@ import XCTest
 class CurrentChatUserAvatarView_Tests: XCTestCase {
     var defaultAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
     var customAvatarView: CustomCurrentChatUserAvatarView! = CustomCurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+    var defaultAppearanceHookAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+
     var mockedCurrentUserController: CurrentChatUserController_Mock<NoExtraData>! = .mock()
 
-//    test_defaultAppearance() ✅
-//    test_contentIsLoaded() // klidne vic casu, treba "s" a "bez" online indicator
-//    test_customDefaultAppearance()
-//    test_customSubclassing() ✅
+    override func setUp() {
+        super.setUp()
+        UIConfig.default.currentUser.currentUserViewAvatarView.defaultAppearance.rules.removeAll()
+    }
 
     func test_customApperanceHook_currentChatUserAvatarView_withImage_looksAsExpected() {
+        UIConfig.default.currentUser.currentUserViewAvatarView.defaultAppearance.addRule {
+            $0.avatarView.imageView.isOpaque = true
+            $0.backgroundColor = .brown
+            $0.avatarView.imageView.backgroundColor = .cyan
+        }
+
         mockedCurrentUserController.currentUser_mock = .init(
             id: "Yoda",
             imageURL: TestImages.yoda.url
         )
-        defaultAvatarView.controller = mockedCurrentUserController
+        defaultAppearanceHookAvatarView.controller = mockedCurrentUserController
 
-        defaultAvatarView.applyDefaultAppearance()
-
-        AssertSnapshot(defaultAvatarView, record: true)
+        AssertSnapshot(defaultAppearanceHookAvatarView)
     }
```

Comment:
Maybe this can be simplified like:
```swift
func test_customizationUsingDefaultAppearanceRule() {
    class TestAvatarView: CurrentChatUserAvatarView {}
    TestAvatarView.defaultAppearance {
        $0.avatarView.imageView.isOpaque = true
        $0.backgroundColor = .brown
        $0.avatarView.imageView.backgroundColor = .cyan
    }

    mockedCurrentUserController.currentUser_mock = .init(
        id: "Yoda",
        imageURL: TestImages.yoda.url
    )

    let view = TestAvatarView()
    view.controller = mockedCurrentUserController

    AssertSnapshot(view, record: true)
}
```

Reply:
Thanks :)